src/vnsw/agent/oper/interface.cc

```diff
@@ -469,7 +469,38 @@ NET_LUID GetInterfaceLuidFromName(const std::string& name, const Interface::Type
     return intf_luid;
 }
 
-NET_IFINDEX GetInterfaceIndexFromLuid(NET_LUID intf_luid) {
+Interface::IfGuid GetInterfaceGuidFromLuid(const NET_LUID intf_luid) {
+    GUID intf_guid;
+
+    NETIO_STATUS status = ConvertInterfaceLuidToGuid(&intf_luid, &intf_guid);
+    if (status != NO_ERROR) {
+        LOG(ERROR, "ERROR: on converting LUID to GUID:" << status);
+        assert(0);
+    }
+
+    Interface::IfGuid result;
+    memcpy(&result, &intf_guid, sizeof(intf_guid));
+
+    return result;
+}
+
+NET_LUID GetInterfaceLuidFromGuid(const Interface::IfGuid intf_guid) {
```

Comment:
I think it would be better to pass GUID by const reference instead of by const value